src/main/resources/db/migration/common/V1_296__add_users_and_timestamps_to_automated_tasks.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE darts.automated_task
+SET created_ts = now(), last_modified_ts = now(), created_by = 0, last_modified_by = 0
+WHERE true;
```

Comment:
You can remove the `WHERE` clause, it's not needed.

Reply:
I got errors when running without a where clause, this was directly against postgres so assumed it would be the case when part of a migration.  Unless it was a limitation imposed by datagrip

Reply:
I tried running one on pgadmin and it worked so possibly.